### DISCUSSION TITLE
Cura Profile for this awesome firmware?

### DISCUSSION THREAD

**Discussion by fysis88** [mriscoc/Ender3V2S1#1262] [Q&A]
Greeting to all the community members.
I own an Ender 3 V2 , with BLT and double Z axis.
My machine had been in the dust for over 1.5 year....Until one week ago that i stepped on this awesome software.
I have done the basic calibration steps , my PLA filaments are over 2 years old , and this printer , does the best job of his life :D 
My question , is there any Cura profile  , with optimized settings for Ender 3 V2 and this excellent firmware?
I am glad i am back to the 3d printing community...
Thank you!

**Comment by csanii123** [comment]
I have been using CHEP's profiles and they work well for me on my stock E3 v2 + mrisoc.
